### ISSUE TITLE
Can't connect to poe.trade :/

### ISSUE THREAD

**Issue by KheopsGaming** [PoE-TradeMacro/POE-TradeMacro#399]
Since I made the update i got these messages : 

http://prntscr.com/g2ilel
http://prntscr.com/g2ilmy

please help me :s

**Comment by Eruyome** [comment]
Please check if you have any anitvir software that blocks `lib\curl.exe`.

**Comment by KheopsGaming** [comment]
I don't have any antivir software :/

**Comment by Eruyome** [comment]
That's something you should fix, not having any antivir software...
Could you open a windows terminal/console in the `lib` folder and execute this? Or just open it anywhere and use the full path to the exe:
`curl.exe -I poe.trade`

You should get an output like this:

![grafik](https://user-images.githubusercontent.com/16058242/28781631-f75da5b4-760a-11e7-96ab-ed4aae95750e.png)

One reason I found for your error was the the curl.exe was being blocked.

**Comment by KheopsGaming** [comment]
I feel like really noob but i don't know how to open a windows terminal/console in the lib folder :s

**Comment by Eruyome** [comment]
https://superuser.com/questions/339997/how-to-open-a-terminal-quickly-from-a-file-explorer-at-a-folder-in-windows-7

Or open it anywhere, by typing `cmd` in the windows search for example and type/paste the full path to the curl.exe.

**Comment by KheopsGaming** [comment]
Huh for no reason i cant open with Shift+right click, i don't have "open command window here". Thank you for your patience, I'll try to find out why its not working

**Comment by KheopsGaming** [comment]
Oh i see command window was powershell on w10 .. but atleast i got error message :/ http://prntscr.com/g2njst

**Comment by Eruyome** [comment]
Wow, didn't even know that.... and that's probably the issue. Does `.\curl.exe -I poe.trade` work?

**Comment by dein0s** [comment]
[Win+R] -> type `cmd`
In the console window type `cd %USERPROFILE%\Documents\POE-TradeMacro-master\lib` to navigate to the TM folder, and then just as was mentioned above `culr.exe -I poe.trade`

And yeah, in PowerShell you have to prefix file names file with `.\` which indicates the current directory in the path.

**Comment by KheopsGaming** [comment]
I don't know what i doing wrong but i reach the lib folder but can't launch culr . exe
http://prntscr.com/g2xqm9

**Comment by KheopsGaming** [comment]
Sorry for french W10 btw :/

**Comment by Eruyome** [comment]
In your case try this
```
"C:\Users\El Poyo\Documents\POE-TradeMacro-master\lib\curl.exe" -I poe.trade
```

Open the console, doesn't matter if in the lib directory or not and paste the entire line.

**Comment by dein0s** [comment]
@KheopsGaming sorry, i've made a typo in a filename :|

**Comment by KheopsGaming** [comment]
He ask me with what i want to open it i select Notepad and he say me that : 
Utilisation : fsutil behavior query <option>

<option>

AllowExtChar
BugcheckOnCorrupt
Disable8dot3 [<Chemin d'accŠs au volume>]
DisableCompression
DisableCompressionLimit
DisableDeleteNotify [NTFS|ReFS]
DisableEncryption
DisableFileMetadataOptimization
DisableLastAccess
DisableSpotCorruptionHandling
EncryptPagingFile
MemoryUsage
MftZone
QuotaNotify
SymlinkEvaluation
DisableWriteAutoTiering [<Chemin d'accŠs au volume>]
DisableTxf [<Chemin d'accŠs au volume>]

Pour plus d'informations, utilisez ® fsutil 8dot3name query /? ¯.

Pour DisableDeleteNotify, si ni NTFS, ni ReFS n'est sp‚cifi‚, les deux valeurs sont affich‚es.

**Comment by KheopsGaming** [comment]
Well i open it in powershell and gimme the same http://prntscr.com/g31c2u

**Comment by Eruyome** [comment]
You're being asked to choose a program to open curl.exe?

**Comment by dein0s** [comment]
It wouldn't work in powershell like this from what I see on the screenshot. It's a total mess.
I can suggest you following:
- create an empty text file anywhere on your pc
- place this string into it `"%USERPROFILE%\Documents\POE-TradeMacro-master\lib\curl.exe" -I poe.trade >> outresult.txt` and save the file
- change the file extension to `bat`
- run it, on success it will create another file `outresult.txt` right next to your bat-file
- copy-paste text from `outresult.txt` in comments here

**Comment by Eruyome** [comment]
So, what did you do to make it work? And does the macro work now, too?

**Comment by KheopsGaming** [comment]
Macro still don't work and for the result i made the file.txt . Btw i still can't connect poe trade :'(

**Comment by Eruyome** [comment]
I want to know if you did anything else besides using the code provided by dein0s, before you were asked what to use to to open the exe file, if that also happens in the background when using the maco it obviously fails. Try to be more detailed in your asnwers, I'll be on vacation in 24hours for a few days, unfortunately. Meaning, you won't get further support for a while then.

**Comment by KheopsGaming** [comment]
Sorry have a lot of work atm :/ And when powershell ask me to open i just used notepad and do nothing after this. I do what denios ask me to do and just try after to launch poe trade helper after but still don't work :s

**Comment by Eruyome** [comment]
Have you taken a look at the link I posted? http://www.askvg.com/solution-exe-files-always-open-with-notepad-or-other-applications/

I'm not sure if you have the described issue but maybe it helps.

**Comment by KheopsGaming** [comment]
I tried this but still don't work :s 
Thanks for all this time you have me devote i download version 2.3.4 (hoping to be able to select the league 3.0 at its release). I wish you a good holiday and regarding my problem I just hope it will settle in future updates
Btw i'm really sorry for this English i use a translator

**Comment by Hiyeh** [comment]
Hey ive also got the same problem and i have manged to follow up to curl.exe -I poe.trade part and this is what i got https://prnt.sc/g446rx.

Tried opening macro up agin but still cant connect https://prnt.sc/g4466w 

what do i do now?

**Comment by Eruyome** [comment]
Use 2.3.4 for now until I have the time to look at this and find the issue.

**Comment by Palaryel** [comment]
After checking all the possible issues in the FAQ, and verifying its not related to any of the network settings I digged a bit deeper, and found something really interesting:

> The problem does only if the path to the macro folder contains spaces. After I copied the macro folder to the root directory (c:\ in my case) and made sure the macro folder itself doesnt have any spaces, the script started working correctly without any errors.

**Comment by KheopsGaming** [comment]
@Palaryel U ARE THE GOD <3 love u so much

**Comment by Eruyome** [comment]
@Palaryel nice find, maybe I forgot to wrap some path in quotation marks somewhere.

**Comment by Eruyome** [comment]
The newest release fixes this (2.3.8).
